ccd-definition/CaseEventToComplexTypes/DefendantResponseLRspec-base-nonprod.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
youll need to remove the `who`'s looking at your story, you might need another file structure jig around or if youre using this one as the one for HNL then remove it from here